packaging/installer/methods/source.md

```diff
@@ -31,14 +34,18 @@ Additionally, the following build time features require additional dependencies:
     -   OpenSSL 1.0.2 or newer _or_ LibreSSL 3.0.0 or newer.
 -   dbengine metric storage:
     -   liblz4 r129 or newer
-    -   OpenSSL 1.0 or newer (LibreSSL _amy_ work, but is largely untested).
+    -   OpenSSL 1.0 or newer _or_ LibreSSL 3.0.0 or newer.
     -   [libJudy](http://judy.sourceforge.net/)
 -   Netdata Cloud support:
     -   A working internet connection
     -   A recent version of CMake
     -   OpenSSL 1.0.2 or newer _or_ LibreSSL 3.0.0 or newer.
     -   JSON-C (may be provided by the user as shown below, or by the system)
     -   protobuf (Google Protocol Buffers) and protoc compiler
+-   Machine Learning support:
+    -   A working C++ compiler
+-   eBPF monitoring support on Linux:
+    -   libelf 0.0.6 or newer (for legacy eBPF) _or_ libelf 0.5 or newer (for CO-RE)
 
 ## Preparing the source tree
 
```

Comment:
Since I can't add comments for lines that didn't change, here's my feedback for lines 52-55

"Certain Netdata features require ..."
"You need to copy the libraries and their header files to specific locations in the source tree you want to use"
--> Not too happy with "specific locations" 
"You can copy these files using `git checkout`. "

(And then the content from line 57 onwards)

Reply:
Agreed on the first point.

Regarding the second one, the choice of wording there is very intentional. The files need to be in those _exact_ locations, otherwise they won’t get used. The use of ‘specific locations’ was intended to emphasize this without getting into a needlessly verbose explanation.

On the third point, the section below is about specific things that need done if working from a git checkout, not anything to do with copying specific dependencies. I think the best approach here is actually to move that section up above this one and put it at H2 level instead of H3.